### DISCUSSION TITLE
Unable run the quack

### DISCUSSION THREAD

**Discussion by hardi-yansah** [greatbit/quack#264] [General]
Hi I'm sorry newbie to use this project, while I run in docker from this repo and finished. Then go to localhost just showing:
![image](https://github.com/user-attachments/assets/17b6282a-2700-472d-af85-315f96e4330e)
Maybe there are other configuration missed.
Thank you

**Comment by azee** [comment]
I believe you are running Apache server on your machine. QuAck uses Jetty. If you'll look into your docker container logs, you'll probably see that the startup failed due to the conflict on the port 80.
Find out what PID is using port 80 and stop it.
Alternatively, you can start quack using docker run command and override the port mapping using -p option - see the example in the README.
